modules/installation-configuration-parameters.adoc

```diff
@@ -2185,6 +2185,12 @@ Additional GCP configuration parameters are described in the following table:
 |The name of the GCP project where the installation program installs the cluster.
 |String.
 
+|platform:
+  gcp:
+    userProvisionedDNS:
+|Enables user-provisioned DNS instead of the default cluster-provisioned DNS solution. If you use this feature, you must provide your own DNS solution that includes records for `api.<cluster_name>.<base_domain>` and `*.apps.<cluster_name>.<base_domain>`.
```

Comment:
To be exact, generally the base domain doesn't have the ending dot character, e.g. the `baseDomain` in install-config. So the exact DNS records would be like `api.<cluster_name>.<base_domain>.` and `*.apps.<cluster_name>.<base_domain>.` respectively.